swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -1,67 +1,137 @@
+| boolean        | <code>configurable boolean isAdmin = ?; </code>                                                                                                                                       | `$ bal run -- -CisAdmin=true` <br> or <br> `$ bal run -- -CisAdmin=1` |
+| int, byte      | <code>configurable byte age = ?; </code><br> <code>configurable int port = ?;</code>                                                                                                  | `$ bal run -- -Cage=25 -Cport=9090`                                 |
+| float, decimal | <code>configurable float height = ?; </code><br> <code>configurable decimal salary = ?;</code>                                                                                        | `$ bal run -- -Cheight=5.6 -Csalary=50500.65`                       |
+| string         | <code>configurable string name = ?; </code>                                                                                                                                           | `$ bal run -- -Cname=John`                                          |
+| xml            | <code>configurable xml book = ?; </code>                                                                                                                                              | `$ bal run -- -CxmlVar="<book>The Lost World</book>"`               |
+| enum           | <code>enum Country { </code><br>    <code>LK = "Sri Lanka", </code><br>    <code>US = "United States" </code><br> <code>} </code><br> <code>configurable Country country = ?; </code> | `$ bal run -- -Ccountry="Sri Lanka"`                                |
+| union          | <code>configurable float&#124;int&#124;string measurement = ?; </code>                                                                                                                | `$ bal run -- -Cmeasurement=5.0`                                    |
+
+## Provide via configuration files
+
+You can provide configuration values via one or more configuration files in which the content is expected to comply with the [TOML syntax](#toml-syntax). 
+
+1. Specify the path of the configuration file via the `BAL_CONFIG_FILES` environment variable. 
+
+2. Ballerina also supports specifying multiple configuration files using the `BAL_CONFIG_FILES` environment variable with the OS-specific separator. The file precedence order will be as specified in the environment variable.
+
+3. If an environment variable is not specified, a file named `Config.toml` will be sought in the current working directory, and this will be used by default when a Ballerina program is executed using the `bal run` command.
+
+    >**Note:**  Once the environment variable is specified, the `Config.toml` will not be considered for the configuration values by default. Therefore, if you are required to use the `Config.toml` file along with others, you need to specify all of them via `BAL_CONFIG_FILES` in the order in which they should be executed. 
```

Comment:
Since point 3 is about `If an environment variable is not specified,` having the details of the exact opposite of it under notes seems not correct. Can we move the `Note` to the end of all the four points?

Reply:
Moved.